File: Modul 5/Modul 5 Bahasa Python/PRAK503-2410817310007-MuhammadNaufalAbdillah.py
def maximal(a, b):
    maxim = a
    if b>maxim:
        maxim=b
    return maxim

def minimal(a, b):
    minim = a
    if b<minim:
        minim=b
    return minim

batas = 0
maxim = -100000
minim = 100000
bilangan = int(input())
nilai = list(map(int, input().split()))
while batas < bilangan:
    nilai1 =nilai[batas]
    maxim = maximal(maxim, nilai1)
    minim = minimal(minim, nilai1)
    batas = batas + 1
print(maxim, minim)

File: Modul 5/Modul 5 Bahasa C/PRAK503-2410817310007-MuhammadNaufalAbdillah.c
#include<stdio.h>
int maxim(int a, int b){
    if (a>b)
        return a;
    else
        return b;
   
}
int minim(int a, int b){
    if (a<b)
        return a;
    else  
        return b;
    
}

int main(){
    int batas=0;
    int maks = -100000;
    int mins = 100000;
    int bilangan;
    scanf("%d", &bilangan);

    while (batas<bilangan)
    {
        int nilai;
        scanf("%d", &nilai);
        maks=maxim(maks,nilai);
        mins=minim(mins, nilai);
        batas++;
    }
    printf("%d %d", maks, mins);
}